airflow/providers/apache/drill/hooks/drill.py

```diff
@@ -0,0 +1,89 @@
+#
```

Comment:
Can you create `__init__.py` file in hooks directory?

Reply:
Ah, how did I miss that?  Thanks.